inc/thorvg.h

```diff
@@ -1119,24 +1128,24 @@ class TVG_EXPORT SwCanvas final : public Canvas
      *
      * The buffer of a desirable size should be allocated and owned by the caller.
      *
-     * @param[in] buffer A pointer to a memory block of the size @p w x @p h, where the raster data are stored.
+     * @param[in] buffer A pointer to a memory block of the size @p stride x @p h, where the raster data are stored.
```

Comment:
in general the buffer size is stride x h, right?

Reply:
@mgrudzinska correct.